website/docs/add-secure-apps/flows-stages/stages/authenticator_endpoint_gdtc/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Endpoint Authenticator Google Device Trust Connector Stage
+---
+
+<span class="badge badge--primary">Enterprise</span>
+<span class="badge badge--version">authentik 2024.10+</span>
+
+---
+
+With this stage, authentik can validate users' Chrome browsers and ensure that users' devices are compliant and up-to-date.
+
+:::info
+This stage only works with Google Chrome, as it relies on the [Chrome Verified Access API](https://developers.google.com/chrome/verified-access).
+:::
+
+## Configuration
+
+The main steps to set up your Google workspace are as follows:
+
+1. [Create your Google Cloud Project](#create-a-google-cloud-project)
+2. [Create a service account](#create-a-service-account)
+3. [Set credentials for the service account](#set-credentials-for-the-service-account)
+4. [Define access and scope in the Admin Console](#set-credentials-for-the-service-account)
+
+For detailed instructions, refer to Google documentation.
+
+### Create a Google cloud project
+
+1. Open the Google Cloud Console (https://cloud.google.com/cloud-console).
+2. In upper left, click the drop-down box to open the **Select a project** modal box, and then select **New Project**.
+3. Create a new project and give it a name like "authentik GWS"
+4. Use the search bar at the top of your new project page to search for "API Library".
+5. On the **API Library** page, use the search bar again to find "Chrome Verified Access API".
+6. On the **Chrome Verified Access API** page, click **Enable**.
+
+### Create a service account
+
+1. After the new Chrome Verified Access API	 is enabled (it might take a few minutes), return to the Google Cloud console home page (click on **Google Cloud** in upper left).
```

Comment:
```suggestion
1. After the new Chrome Verified Access API is enabled (it might take a few minutes), return to the Google Cloud console home page (click on **Google Cloud** in upper left).
```

Reply:
There was weird spacing here @BeryJu you might want to double-check no other words went missing.